formio_support/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+{
+    'name': 'Forms | Ticket',
+    'summary': 'Forms integration with Support Ticket',
+    'version': '12.0.1.0.0',
+    'license': 'LGPL-3',
+    'author': 'MultidadosTI',
+    'website': 'www.multidadosti.com.br',
+    'category': 'Project',
```

Comment:
```suggestion
    'category': 'Others',
```
O modulo em si nao tem relação com projeto